.circleci/config.base.yml

```diff
@@ -16,7 +16,7 @@ parameters:
 executors:
   w_medium: &windows-e2e-executor-medium
     machine:
-      image: 'windows-server-2019-vs2019:stable'
+      image: 'windows-server-2022-gui:current'
```

Comment:
I was experimenting with installing Node 18 on old image too.
However I was running into issues with older version of nvm and conflicts with pre-installed node 14.
New image comes with fixed version of nvm but without node nor npm nor yarn, but at least installing them works as expected.